java/src/org/certificatetransparency/ctlog/MerkleAuditProof.java

```diff
@@ -0,0 +1,18 @@
+package org.certificatetransparency.ctlog;
+
+import org.certificatetransparency.ctlog.proto.Ct;
+
+
+public class MerkleAuditProof {
+  public Ct.Version version;
+  public long treeSize;
+  public long leafIndex;
+  public java.util.List<byte[]> pathNode;
+
+  public MerkleAuditProof(Ct.Version version_, long treeSize_, long leafIndex_) {
```

Comment:
this is odd style.  trailing underscores I mean.  you might remove them, and have the lines below read 'this.version = version' which is pretty much how I've seen Java done.